strings/brain_damage_lines.json

```diff
@@ -121,4 +123,17 @@
         "sheadow lings",
         "abdoocters"
     ]
+    "game":[
+		" ",
```

Comment:
still tabs

do you not have a way to view tabs and spaces in your editor?

Reply:
properly fixed it